lib/siwapp_web/live/live_helpers.ex

```diff
@@ -12,4 +12,24 @@ defmodule SiwappWeb.LiveHelpers do
     modal_opts = [id: :modal, return_to: path, component: component, opts: opts]
     live_component(SiwappWeb.ModalComponent, modal_opts)
   end
+
+  def checked?(checked, id) do
+    Enum.member?(checked, id)
+  end
+
+  def status(paid, due_date) do
```

Comment:
This status fn should go to the Invoices context. It's not just a helper for liveview, it's something more internal of every invoice.